node_modules/titanium-sdk/lib/emulators/genymotion.js

```diff
@@ -286,7 +286,10 @@ function findGenymotion(dir, config, callback) {
 	if (!player || !fs.existsSync(player)) {
 		player = path.join(dir, 'player' + exe);
 	}
-	if (player && !fs.existsSync(player)) {
+	if (!fs.existsSync(player) && process.platform != 'win32') {
```

Comment:
I get the darwin fix, but why the truthy check? After line https://github.com/FokkeZB/titanium_mobile/blob/patch-37/node_modules/titanium-sdk/lib/emulators/genymotion.js#L287 it will always be truthy since `path.join()` does not resolve and just joins the arguments right?